sample_data/ingest_sample_data.py

```diff
@@ -46,9 +51,9 @@ def ingest_file(file_info, index=0, dataset=None, chart=None):
             new_file_item.file.save(file_path, ContentFile(f.read()))
 
 
-def ingest_contexts():
+def ingest_contexts(use_case):
     print('Creating Context objects...')
-    with open('sample_data/contexts.json') as contexts_json:
+    with open(str(USE_CASE_FOLDER / use_case / 'contexts.json')) as contexts_json:
```

Comment:
I don't believe `str` is required here. Assuming that's the case, I'd recommend batching this change with the other two spots I suggested the change.
```suggestion
    with open(USE_CASE_FOLDER / use_case / 'contexts.json') as contexts_json:
```

Reply:
Changed in [3eda2f4](https://github.com/OpenGeoscience/uvdat/pull/45/commits/3eda2f4140862f1322782ad15956d7c9eac41493)